packages/integration-tests/suites/public-api/startTransaction/basic_usage/test.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from '@playwright/test';
+import { Event } from '@sentry/types';
+
+import { sentryTest } from '../../../../utils/fixtures';
+import { getSentryTransactionRequest } from '../../../../utils/helpers';
+
+sentryTest('should report a transaction in an envelope', async ({ getLocalTestPath, page }) => {
+  const url = await getLocalTestPath({ testDir: __dirname });
+  const eventData = await getSentryTransactionRequest(page, url);
+  const transaction = eventData[2] as Event;
+
+  expect(transaction.transaction).toBe('test_transaction_1');
+  expect(transaction.spans).toBeDefined();
+});
+
+sentryTest('should report finished spans as children of the root transaction', async ({ getLocalTestPath, page }) => {
+  const url = await getLocalTestPath({ testDir: __dirname });
+  const eventData = await getSentryTransactionRequest(page, url);
+  const transaction = eventData[2] as Event;
```

Comment:
We should be returning the correct type from the helpers.

`eventData[0]` and `eventData[1]` are probably sessions. We prob wanna filter those out in the helper's return value